.github/workflows/test.yaml

```diff
@@ -287,25 +285,6 @@ jobs:
           go-version: '1.22'
           cache: true
       - run: go get
-      - run: |
```

Comment:
Tried to simplify this but lmk if it's in the wrong direction @dhiltgen

Reply:
While this does simplify the yaml, it increases build times because more variants are being built.  Given the filter logic we've added to do a full generate if anything in the ./llm dir is touched, I think we should try to keep this lean so CI runs faster.